bundles/org.eclipse.equinox.p2.director.app/META-INF/MANIFEST.MF

```diff
@@ -21,14 +21,12 @@ Import-Package: org.bouncycastle.openpgp,
  org.eclipse.equinox.internal.provisional.p2.director,
  org.eclipse.equinox.internal.provisional.p2.repository,
  org.eclipse.equinox.p2.core;version="[2.8.0,3.0.0)",
- org.eclipse.equinox.p2.core.spi;version="[2.1.0,3.0.0)",
  org.eclipse.equinox.p2.engine;version="[2.3.0,3.0.0)",
  org.eclipse.equinox.p2.engine.query;version="[2.0.0,3.0.0)",
  org.eclipse.equinox.p2.metadata;version="[2.0.0,3.0.0)",
  org.eclipse.equinox.p2.metadata.expression;version="2.0.0",
  org.eclipse.equinox.p2.planner;version="[2.0.0,3.0.0)",
  org.eclipse.equinox.p2.query;version="[2.0.0,3.0.0)",
- org.eclipse.equinox.p2.repository;version="[2.0.0,3.0.0)",
```

Comment:
This causes access restriction warnings. Need to check if its a JDT issue maybe.